recipes/Blender.yml

```diff
@@ -0,0 +1,26 @@
+app: Blender
+
+ingredients:
+  script:
+    - wget -c https://builder.blender.org/download/blender-2.80-91a155833e59-linux-glibc224-x86_64.tar.bz2
```

Comment:
Please do not hardcode the payload application version. Rather, parse it from the download page. See 

https://github.com/AppImage/pkg2appimage/blob/3168d7ce787246feb697a950005fbffec0533def/recipes/CAMotics.yml#L20-L21

for example.

Reply:
Sure, I'll continue to play with it on the weekend. 

Any recommendations on how to deal with download file redirection? I've been having a tough time dealing with that when using wget.

Reply:
Please try my edited version. Thanks.

Reply:
Ok I'll give that a try later tonight.